src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -724,7 +727,7 @@
     <HelixCorrelationPayload Include="$(CoreRootDirectory)" />
 
     <!-- Browser-Wasm follows a very different workflow, which is currently out of scope of the Log Checker. -->
-    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" Condition="'$(TargetsBrowser)' != 'true'" />
+    <HelixCorrelationPayload Include="$(XUnitLogCheckerDirectory)" Condition="'$(TargetsBrowser)' != 'true' and '$(TargetOS)' != 'iOS' and '$(TargetOS)' != 'iOSSimulator' and '$(TargetOS)' != 'tvOS' and '$(TargetOS)' != 'tvOSSimulator'" />
```

Comment:
nit: Should this also use `TargetsiOS`, `TargetsiOSSimulator`, `TargetstvOS`, `TargetstvOSSimulator` to conform with `TargetsBrowser`

What does the XUnitLogChecker do? Should we update the comment above if Log Checker shouldn't be working on ioslike either?

Reply:
Correct, it's not useful on any platform that uses xharness and it can't be restored.